src/pytest_plugins/filler/filler.py

```diff
@@ -172,6 +173,17 @@ def pytest_addoption(parser: pytest.Parser):
         default=True,
         help="Skip generating an index file for all produced fixtures.",
     )
+    test_group.addoption(
```

Comment:
Having this in the test group feels better than on it's own 👍 Did you happen to understand why the new test group I added wasn't showing up with a unique description?

Reply:
It's a bit of a mess the help in our commands 😅

We do some regex matching to see what to include and what not to include in the commands, and I think some cleanup is necessary because the current method is not ideal.